corehq/apps/locations/tests/test_views.py

```diff
@@ -23,9 +26,11 @@
     'shares_cases': False,
     'view_descendants': False,
     'expand_view_child_data_to': None,
+    'has_users': True,
 }
 
 
+@es_test(requires=[user_adapter])
```

Comment:
Isn't the ES call mocked out?

Reply:
> I thought the best practice would be to mock at the class level and override the return value for one method

That does sound nice, but I believe you that it's hard.  Seems reasonable to do the ES call during this view. I'm not too worried about it